### PR TITLE
New functuin add_header is added

### PR DESCRIPTION
**Pull request by naghmeh-sh**

### CODE REVIEW

File: src/Talis/Html/Form/Validator/Container.php [hunk 1]
```diff
@@ -36,6 +38,15 @@ public function add_required(\Talis\Html\Form\Element\aElement $element,string $
         return $this;
     }
     
+    /**
+     * adds JS header of the validation at the same level of excluded/live/...
+     */
+    public function add_header(string $header_name,string $header_action=''){
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php]: call it add_to_header

File: src/Talis/Html/Form/Validator/Container.php [hunk 2]
```diff
@@ -80,6 +91,10 @@ private function header():\stdClass{
      */
     public function __toString(){
         $full_js = $this->header();
+        foreach($this->headers as $name => $action){
```
Comment (itay-moav) [inline src/Talis/Html/Form/Validator/Container.php]: move this section into the header()

File: src/Talis/Html/Form/Validator/Container.php (hunk 1 above)
Comment (itay-moav) [reply src/Talis/Html/Form/Validator/Container.php]: and do full phpdoc comments

**Review by itay-moav — CHANGES_REQUESTED** [review]

**Comment by naghmeh-sh** [comment]
new changes moved to header

**Review by itay-moav — APPROVED** [review]

**Pull request merged**